install_builder/deadline-cloud-for-nuke.xml

```diff
@@ -18,7 +18,7 @@
                     <onPackingFilterList>
                         <fileNameFilter pattern="*/deadline/nuke_submitter/*" logic="matches" patternType="glob"/>
                         <fileNameFilter pattern="*/deadline/nuke_util/*" logic="matches" patternType="glob"/>
-                        <fileNameFilter pattern="*/menu.py" logic="matches" patternType="glob"/>
+                        <fileNameFilter pattern="*/deadline/nuke_submitter/menu.py" logic="matches" patternType="glob"/>
```

Comment:
Line 19 now seems to have the menu.py covered

Reply:
Make sense. Will remove this in the next revision.